### PR TITLE
Update free-programming-books.md

### PR DESCRIPTION
**Pull request by Devinsuit**

### CODE REVIEW

**Comment by johnaoss** [comment]
I'm not entirely sure what you're aiming to change with your PR. Can you please comment with why you've changed links / removed sources? Thank you!

**Comment by vhf** [comment]
Dead links replacement and removal when no replacement found I guess @johnaoss :) 

I'll take a closer look but I have a lot of trust in @Devinsuit 's quality contributions :+1:

**Comment by johnaoss** [comment]
Ah okay, that makes more sense. Thank you @vhf , was just wondering why they removed the Unix as IDE book when it is still up on GitHub. Copyright maybe?

**Comment by Devinsuit** [comment]
I accidentally removed Unix as IDE. My bad.

File: free-programming-books.md [hunk 1]
```diff
@@ -237,7 +237,7 @@
 * [Learn OpenGL](http://learnopengl.com) - Joey de Vries
 * [Learning Modern 3D Graphics Programming](https://web.archive.org/web/20150225192611/http://www.arcsynthesis.org/gltut/index.html) (draft)
 * [OpenGL](http://www.songho.ca/opengl/index.html) - Concepts and illustrations.
-* [OpenGL Programming Guide (The Red Book)](http://fly.srk.fer.hr/~unreal/theredbook/)
+* [OpenGL Programming Guide (The Red Book)](https://sgar91.files.wordpress.com/2010/12/opengl-programming-guide-7e.pdf)
```
Comment (thebouv) [inline free-programming-books.md]: I think this is the line Travis build is complaining about. Needs PDF indicator.

**Pull request closed without merging**